modules/Attendance/attendance_take_byPerson.php

```diff
@@ -132,7 +132,7 @@
 
                         $output = '<b>'.__($log['direction']).'</b> ('.__($log['type']). (!empty($log['reason'])? ', '.__($log['reason']) : '') .')';
                         if (!empty($log['comment'])) {
-                            $output .= '&nbsp;<img title="'.$log['comment'].'" src="./themes/'.$_SESSION[$guid]['gibbonThemeName'].'/img/messageWall.png" width=16 height=16/>';
+                            $output .= '&nbsp;<img title="'.$log['comment'].'" src="./themes/'.$session->get('gibbonThemeName').'/img/messageWall.png" width=16 height=16/>';
```

Comment:
This is inside a function and will throw an error. Be sure to test these as it'll help you catch these kinds of errors.

Reply:
It doesn't look like this issue has been resolved yet.